source/common/protobuf/utility.cc

```diff
@@ -118,6 +118,12 @@ void MessageUtil::checkForDeprecation(const Protobuf::Message& message, bool war
       continue;
     }
 
+    bool warn_only = true;
+    if (runtime && !runtime->snapshot().deprecatedFeatureEnabled(
+                       absl::StrCat("envoy.deprecated_feature.", field->name()))) {
```

Comment:
Not sure if it worth, perhaps adding a file scope for them? I think you can easily get file info from `field->file()` here and it is easy for grep as well?

Reply:
Actually checking on this I've clearly not been paying sufficient attention.

we've got a bunch of "config" fields across a bunch of files deprecated.  If you think there's value in enabling/disabling them individually rather than en masse, I'll convert it over